bundles/org.openhab.binding.solax/src/main/java/org/openhab/binding/solax/internal/SolaxBindingConstants.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.solax.internal;
+
+import java.util.Set;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link SolaxBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Konstantin Polihronov - Initial contribution
+ */
+@NonNullByDefault
+public class SolaxBindingConstants {
+
+    private static final String BINDING_ID = "solax";
+    private static final String THING_LOCAL_CONNECT_INVERTER_ID = "local-connect-inverter";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID THING_TYPE_LOCAL_CONNECT_INVERTER = new ThingTypeUID(BINDING_ID,
+            THING_LOCAL_CONNECT_INVERTER_ID);
+
+    public static final Set<ThingTypeUID> SUPPORTED_THING_TYPES_UIDS = Set.of(THING_TYPE_LOCAL_CONNECT_INVERTER);
+
+    // List of properties
+    public static final String PROPERTY_INVERTER_TYPE = "inverter-type";
```

Comment:
Properties should be camelCase:
```suggestion
    public static final String PROPERTY_INVERTER_TYPE = "inverterType";
```